brick_library.scm

```diff
@@ -0,0 +1,215 @@
+(brick-library
```

Comment:
It might be nice to invent a new `.tenscript` extension and just tell our IDEs to interpret .tenscript as Scheme. To make it seem more legit :)

Reply:
Good idea